db/sqlx/main.go

```diff
@@ -60,6 +60,22 @@ func main() {
 		}
 		fmt.Printf("ID: %v, FirstName: %v, LastName: %v\n", user.ID, user.FirstName, user.LastName)
 		return
+	case "update":
+		if *id == "" {
+			log.Fatal("You need id.")
+		}
+		if *firstName == "" {
+			log.Fatal("You need a first name.")
+		}
+		if *lastName == "" {
+			log.Fatal("You need a last name.")
+		}
+		err := userController.Update(*id, *firstName, *lastName)
+		if err != nil {
```

Comment:
Goだと以下の様にする事が多いです。
1行になってシンプルになったり、変数のスコープがif文に狭まるのでメモリ効率が良いです。
```suggestion
if err := userController.Update(*id, *firstName, *lastName); err != nil {
```

Reply:
なるほど 👀 
rubyだとifで代入するのは可読性が悪いので設定によってはlinterに怒られたりしますが、ここではerrに代入するだけなのでこの方が良さそうですね